packages/@ember/-internals/metal/lib/events.ts

```diff
@@ -146,11 +146,13 @@ export function sendEvent(
     if (!target) {
       target = obj;
     }
-    if ('string' === typeof method) {
-      method = target[method] as Function;
+
+    let type = typeof method;
+    if (type === 'string' || type === 'symbol') {
+      method = target[method as string] as Function;
     }
 
-    method.apply(target, params);
+    (method as Function).apply(target, params);
```

Comment:
Did we resolve the perf concern here? Should we do this (which is what I think I had suggested/mentioned), or go back to:

```ts
if (typeof method === 'string' || typeof method === 'symbol') {
  method = target[method] as Function;
}
```